### DISCUSSION TITLE
🥰 [FEATURE]空数据时，能否显示一个empty图标？

### DISCUSSION THREAD

**Discussion by qingfengmy** [antvis/G2Plot#3559] [Q&A] [answered]
### 🥰 Features description [Please make everyone to understand it]
数据为空时，显示empty图标
### 🏞 What problem does this feature solve
现在没有数据，大空白太难看了
### 🧐 What does the proposed API look like
empty={<img />}

**Comment by hustcc** [comment, marked as answer]
感谢反馈，我们考虑下这个事情。

不过即时增加，也是在新版本 G2 v5 中添加，然后 G2Plot 升级即可，时间上暂时不还不能保障，所以你这边可以先在业务中判断数据，然后盖一个图标。
